R/page.R

```diff
@@ -161,17 +265,72 @@ page_sidebar <- function(..., sidebar = NULL, title = NULL, fillable = TRUE, fil
 }
 
 
-#' @rdname page
-#' @inheritParams navset_bar
-#' @seealso [shiny::navbarPage()]
+#' Multi-page app with a top navigation bar
+#'
+#' @description
+#' Create a page that contains a top level navigation bar that can be used to
+#' toggle a set of [nav_panel()] elements. Use this page layout to create the
+#' effect of a multi-page app, where your app's content is broken up into
+#' multiple "pages" that can be navigated to via the top navigation bar.
+#'
 #' @param fillable_mobile Whether or not `fillable` pages should fill the viewport's
 #'   height on mobile devices (i.e., narrow windows).
 #' @param underline Whether or not to add underline styling to page links when
 #'   active or focused.
 #' @param window_title the browser window title. The default value, `NA`, means
 #'   to use any character strings that appear in `title` (if none are found, the
 #'   host URL of the page is displayed by default).
+#' @inheritParams navset_bar
+#' @inheritParams page
+#'
 #' @export
+#' @family Dashboard page layouts
+#'
+#' @seealso [nav_panel()], [nav_menu()], and [nav_item()] for adding content
+#'   sections and organizing or creating items in the navigation bar.
+#' @seealso [layout_columns()] and [layout_column_wrap()] for laying out content
+#'   into rows and columns.
+#' @seealso [layout_sidebar()] for 'floating' sidebar layouts, or for details
```

Comment:
This kind of feels like a distraction for `page_navbar()` (especially given there is a `sidebar` argument that points to the same page)

```suggestion
```